### PR TITLE
Rename UserBell and MagicBell classes

### PR DESCRIPTION
**Pull request by josuemontano**
- [x] Rename the `UserBell` class to `User`
- [x] Rename the `MagicBell` class to `MagicBellClient`
- [x] Update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,155 +1,195 @@
...
+To achieve it, extend `MagicBell` and set the instance in your application's delegate (at either the `AppDelegate.swift` or `App.swift` files):
 
-Therefore, you can instantiate it in the `AppDelegate.swift` or `App.swift` file (or any other file) as a constant value. 
+```swift
+import MagicBell
+
+extension MagicBell {
+    static var shared: MagicBellClient!
+}
+```
```
Comment (vilanovi) [inline README.md:132]: Hi @josuemontano,

This is not working on Swift. `MagicBell` is a module and cannot be extended. Still, if it was a class or a struct, you can't extend by adding attributes (you can only add methods).

Therefore, I'd leave it as it was before: a global attribute `let magicBell = MagicBellClient(apiKey: "...")`. It's actually a good approach on Swift, and quite simple to understand.

**Review by vilanovi — CHANGES_REQUESTED** [review]
I've fixed the Example project, as it was not building.

Added one comment on the README file.

File: README.md (hunk 1 above)
Comment (vilanovi) [reply README.md:132]: If you want, I can edit the snippet. Let me know.

Comment (vilanovi) [reply README.md:132]: Also, note the whole README.md file uses in its snippets `magicBell` as the client instance. If we change this approach, we should also change all the snippets to use the default way to access the main instance.

Comment (josuemontano) [reply README.md:132]: @vilanovi What would it take to make `MagicBell` a class or struct that can be extended?

Comment (josuemontano) [reply README.md:132]: @vilanovi What would it take to make `MagicBell` a type that can be extended?

Comment (vilanovi) [reply README.md:132]: We did have it, with the `MagicBell` class, and we removed it in favor of uniquely having named `MagicBell` the module.

Still, an extension cannot contain new properties (static or instance-based), therefore a user could not extend it to add a custom `shared` static property.

**Pull request merged**